### PR TITLE
Support --skip <targets>.

### PR DESCRIPTION
**Pull request by ejball**

### CODE REVIEW

File: src/Faithlife.Build/BuildRunner.cs [hunk 1]
```diff
@@ -42,11 +43,38 @@ public static int Execute(string[] args, Action<BuildApp> initialize)
...
 					bullseyeArgs.Add(c_defaultTarget);
 
+				var skipDependencies = skipDependenciesFlag.Value;
+				if (skipOption.Value is not null && !skipDependencies)
+				{
+					var skipTargetNames = new HashSet<string>(skipOption.Value.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries));
+					skipDependencies = true;
+
+					var targetNames = new HashSet<string>(bullseyeArgs);
+					foreach (var targetName in bullseyeArgs.ToList())
```
Comment (bgrainger) [inline src/Faithlife.Build/BuildRunner.cs]: This is a bit subtle (having `AddTargetDependencies` capture _and mutate_ this collection); I might have moved it to `var originalBullseyeArgs = bullseyeArgs.ToList();` for clarity. Or not mutated the captured local.

**Review by bgrainger — APPROVED** [review]

File: src/Faithlife.Build/BuildRunner.cs (hunk 1 above)
Comment (ejball) [reply src/Faithlife.Build/BuildRunner.cs]: It's true. Since you noticed I was being lazy, I'll fix it up.

Comment (ejball) [reply src/Faithlife.Build/BuildRunner.cs]: Reorganized the code. Any better?

**Review by bgrainger — APPROVED** [review]
This is a lot clearer, thanks!

**Pull request merged**